.claude/skills/konflux-onboarding/SKILL.md

```
@@ -0,0 +1,278 @@
+   ```bash
+   if [ -d "packages/$COMPONENT_NAME" ]; then
+     TYPE="A"
+   elif [ -d "$COMPONENT_NAME" ] && [ -f "$COMPONENT_NAME/go.mod" ]; then
+     TYPE="B"
+   else
+     # Ask user to clarify
+   fi
+   ```
+
+3. Detect existing state:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)